cmd/ddev/cmd/delete-images.go

```diff
@@ -39,6 +45,23 @@ ddev delete images -y`,
 		if err != nil {
 			util.Failed("Failed to list images: %v", err)
 		}
+
+		// The user can select to delete all ddev images.
+		if deleteAllImages {
+			// Attempt to find ddev images by tag, searching for "drud/ddev-".
+			// Some ddev images will not be found by this tag, future work will
+			// be done to improve finding database images.
+			for _, image := range images {
+				for _, tag := range image.RepoTags {
+					if strings.HasPrefix(tag, "drud/ddev-") {
+						dockerRemoveImage(tag)
```

Comment:
This doesn't remove Postgres or Busybox or Phpmyadmin images. I think it should be improved to do so.